tools/versionbump/versionbump.go

```diff
@@ -0,0 +1,83 @@
+package main
```

Comment:
We have to tell the tagger what version to tag. `tito` has no utilities for bumping versions AFAICT

/cc @dgoodwin